HackerRank/Plus Minus/main.js

```diff
@@ -0,0 +1,24 @@
+function plusMinus(arr) {
+  let n = arr.length;
+  let positive = [];
+  let zero = [];
```

Comment:
its better to use const here, because you are pushing to the arra not setting value to it, so you are not doing change on the value